### PR TITLE
Additional migration considerations from IRL testing.

### PR DESCRIPTION
**Pull request by litzell**
Adding another batch of 4.3 to 6 migration considerations. 

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpdb-doc/dita/install_guide/migrate.xml [hunk 1]
```diff
@@ -129,9 +139,14 @@
               type is changed to <codeph>int2vector</codeph>. A backend function
                 <codeph>pg_get_table_distributedby()</codeph> was added to get the distribution
               policy for a table as a string.</li>
-            <li>In the <codeph>session_level_memory_consumption</codeph> system view, the
-                <codeph>__gp_localid</codeph> and <codeph>__gp_masterid</codeph> columns are
-              removed.</li>
+            <li>The <codeph>__gp_localid</codeph> and <codeph>__gp_masterid</codeph> columns are
+              removed from the <codeph>session_level_memory_consumption</codeph> system view in
+              Greenplum 6. The underlying external tables and functions are removed from the
+                <codeph>gptoolkit</codeph> schema.</li>
```
Comment (mkiyama) [inline gpdb-doc/dita/install_guide/migrate.xml]: gptoolkit -> gp_toolkit

File: gpdb-doc/dita/install_guide/migrate.xml [hunk 2]
```diff
@@ -129,9 +139,14 @@
...
+            <li>The <codeph>CREATE EXTERNAL TABLE</codeph> clause <codeph>INTO
+                  <varname>error_table</varname></codeph> was deprecated in Greenplum 4.3 and is
```
Comment (mkiyama) [inline gpdb-doc/dita/install_guide/migrate.xml]: Specifying an "error table" feature was also removed for COPY and gpload.
The errors are stored in GPDB for 6.x for all three, CREATE EXTERNAL TABLE, COPY, gpload.

I think those are the only three that have that feature.

File: gpdb-doc/dita/install_guide/migrate.xml [hunk 3]
```diff
@@ -197,6 +224,11 @@
           lists the duplicate keys in the <codeph>DISTRIBUTED BY</codeph> clause. Restoring this
           backup will cause a duplicate distribution key error. The <codeph>CREATE TABLE AS</codeph>
           command was fixed in Greenplum 5.10 to disallow duplicate distribution keys.</li>
+        <li>Greenplum 4.3 supports foreign key constraints on columns of different types, for
+          example, <codeph>numeric</codeph> and <codeph>bigint</codeph>, with implicit type
+          conversion. Greenplum 5 and 6 do not support implicit type conversion. Restoring a table
+          with a foreign key on columns with different data types causes an error causes an
+          error.</li>
```
Comment (dyozie) [inline gpdb-doc/dita/install_guide/migrate.xml]: Repeated words here:  causes an error causes an error

**Comment by dyozie** [comment]
@oakbarrett @kyeap-pivotal Any final thoughts on this before we merge?

**Pull request merged**